src/main/java/com/iexec/worker/tee/post/compute/App.java

```diff
@@ -22,11 +27,35 @@ private App() {
     public static void main(String[] args) {
         log.info("Tee worker post-compute started");
 
+        String taskId = null;
+        Optional<ReplicateStatusCause> statusCause = Optional.empty();
+
+        try {
+            taskId = EnvUtils.getEnvVarOrThrow(RESULT_TASK_ID, POST_COMPUTE_MISSING_TASK_ID);
+            runPostCompute(taskId);
+        } catch(PostComputeException e) {
+            log.error("TEE post-compute failed with a known error " +
+                            "[errorMessage:{}]",
+                    e.getStatusCause(), e);
+            statusCause = Optional.of(e.getStatusCause());
+        } catch (Exception e) {
+            log.error("TEE post-compute failed with an unknown error", e);
+            statusCause = Optional.of(POST_COMPUTE_UNKNOWN_ISSUE);
+        } finally {
+            if (statusCause.isPresent()) {
+                sendExitMessageToHost(taskId, statusCause.get());
+            }
+
+            log.info("TEE post-compute finished");
```

Comment:
I would display different outputs whether we have a success or not, something like:
```
log.info(cool)
log.error(notcool)
```

Reply:
In case there's an issue, we already display something like:
```
TEE post-compute failed with a known error 
```
before the `TEE post-compute finished` message. I tend to think it is sufficient.